pkg/controller/petset/pet.go

```diff
@@ -297,22 +297,23 @@ type petHealthChecker interface {
 // It doesn't update, probe or get the pod.
 type defaultPetHealthChecker struct{}
 
-// isHealthy returns true if the pod is running and has the
-// "pod.alpha.kubernetes.io/initialized" set to "true".
+// isHealthy returns true if the pod is running and if it has the
+// "pod.alpha.kubernetes.io/initialized" annotation, it is set to "true".
 func (d *defaultPetHealthChecker) isHealthy(pod *api.Pod) bool {
 	if pod == nil || pod.Status.Phase != api.PodRunning {
 		return false
 	}
 	initialized, ok := pod.Annotations[PetSetInitAnnotation]
 	if !ok {
-		glog.Infof("PetSet pod %v in %v, waiting on annotation %v", api.PodRunning, pod.Name, PetSetInitAnnotation)
-		return false
+		// if the annotation was not found, we just perform a readiness check.
```

Comment:
nit: we don't perform the check, we just check to see if the pod has passed the check